oggm/tests/test_models.py

```diff
@@ -388,6 +407,67 @@ def test_prcp_fac_update(self, hef_gdir):
                                              check_calib_params=True)
         with pytest.raises(InvalidWorkflowError):
             mb_mod.prcp_fac = 10
+        with pytest.raises(InvalidWorkflowError):
+            mb_mod.temp_bias = 1
+
+        # normally this should also be implemented for mu_star ...
+        # with pytest.raises(InvalidWorkflowError):
+        #    mb_mod.mu_star = 100
+
+    def test_prcp_fac_temp_bias_update_constant_mb(self, hef_gdir):
```

Comment:
I just added this to check if it also works for ConstantMassBalance because I thought this could be the error for the UncertainMassBalance but it works normally, this can be also removed again